schema.graphql

```diff
@@ -800,6 +800,61 @@ type NablaSwapFee @entity {
     backstopPool: BackstopPool # link to the BackstopPool entity
 }
 
+type NablaSwap @entity {
+    id: ID! # calculated from blockNumber - extrinsicIndex
+    timestamp: DateTime! # need this to pull recent txns for specific token or pair
+    # from router Swap event
+    sender: String!
+    amountIn: BigInt!
+    amountOut: BigInt!
+    tokenIn: String!
+    tokenOut: String!
+    to: String!
+
+    # this will bring backstop pool and swap pool info
+    swapFee: NablaSwapFee
+}
+
+type NablaBackstopLiquidityDeposit @entity {
+    id: ID! # calculated from blockNumber - extrinsicIndex
+    timestamp: DateTime!
+
+    # from backstop Mint event
+    sender: String!
+    poolSharesMinted: BigInt!
+    amountPrincipleDeposited: BigInt!
```

Comment:
The notion `principle` is taken from the Nabla event definition. "Principle" was the old word the Nabla team used for pool token. This is still used in the `Mint` and `Burn` event definitions.

You might want to call this `amountPoolTokensDeposited/...Withdrawn` in this and the following entity definitions instead.

Reply:
Done